src/views/simplified-account/settings/webhooks/create.njk

```diff
@@ -1,10 +1,68 @@
 {% extends "../settings-layout.njk" %}
 
-{% block settingsPageTitle %}
-  Create a webhook
-{% endblock %}
+{% set settingsPageTitle = "Webhook details" %}
 
 {% block settingsContent %}
-  <h1 class="govuk-heading-l">Webhook details</h1>
+  <h1 class="govuk-heading-l">{{ settingsPageTitle }}</h1>
+  <form method="post" novalidate>
+    <input id="csrf" name="csrfToken" type="hidden" value="{{ csrf }}"/>
+
+    {% set subscriptionItems = [] %}
+    {% for key, name in eventTypes %}
+      {% set normalisedKey = key | lower %}
+      {% set subscriptionItems = (subscriptionItems.push({ value: normalisedKey, text: name }), subscriptionItems) %}
+    {% endfor %}
+
+    {{ govukInput({
+      label: {
+        text: "Callback URL",
+        classes: "govuk-label--s"
+      },
+      hint: { text: "The HTTPS URL GOV.UK Pay will send webhook messages to" },
+      id: "callback_url",
+      name: "callback_url",
+      value: form.values.callback_url,
+      type: "url",
+      spellcheck: false,
+      errorMessage: form.errors.callback_url and {
+        text: form.errors.callback_url
```

Comment:
Can the uses of `callback_url` be camelCase instead? (with the exception of the ID, which may need to be kebab-case for the validation logic)

Reply:
ah, yes, I've now removed these errorMessage and value properties because they aren't needed for the purpose of this PR, I will add those in later.